src/Button/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./Button";
```

Comment:
What's the advantage of this? Wouldn't it be simpler to just rename the `button.tsx` file to `index.tsx`?

Reply:
`Button/index.tsx` leads to having multiple index files => grepping / context becomes murky. I also think this more declaritive structure would scale better. This could be just my opinion, open to changing back.

Reply:
Why can also just put the contents of `button.tsx` into `index.tsx`